avm/res/search/search-service/main.bicep

```diff
@@ -91,11 +94,13 @@ param semanticSearch string?
 ])
 param sku string = 'standard'
 
+import { managedIdentityAllType } from 'br/public:avm/utl/types/avm-common-types:0.3.0'
 @description('Optional. The managed identity definition for this resource.')
-param managedIdentities managedIdentitiesType
+param managedIdentities managedIdentityAllType?
 
+import { diagnosticSettingFullType } from 'br/public:avm/utl/types/avm-common-types:0.2.1'
```

Comment:
Any specific reason for not using the latest version here?

Reply:
@ReneHezser Thanks for pointing this out, no, just a copy/paste error. Fixed: https://github.com/Azure/bicep-registry-modules/pull/3767/commits/0f98fc429caddb6fd90d5489938fbc6f32ac4aa6

Reply:
@ReneHezser test successful.